tests/foreman/cli/test_import.py

```diff
@@ -0,0 +1,110 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Host Collection CLI"""
+from ddt import ddt
+from fauxfactory import gen_string
+from robottelo.common import ssh
+from robottelo.common.helpers import prepare_import_data
+from robottelo.test import CLITestCase
+from robottelo.cli.import_ import Import
+from robottelo.cli.org import Org
+
+
+@ddt
+def csv_to_dataset(csv):
+    """Converts the csv with header as a first entry to a python dict"""
+    keys = csv[0].split(',')
+    del csv[0]
+    dataset = list()
+    for val in csv:
+        dataset.append(
+            dict(zip(keys, val.split(',')))
+        )
+    return dataset
+
+
+class TestImport(CLITestCase):
+    """Import CLI tests.
+
+    All default tests pass no options to the imprt object
+    In such case methods download a default data set from URL
+    specified in robottelo.properties.
+
+    """
+    def test_import_orgs_default(self):
+        """@test: Import all organizations from the default data set
+        (predefined source).
+
+        @feature: Import Organizations
+
+        @assert: 3 Organizations are created
+
+        """
+        files = prepare_import_data()[1]
+        rc1 = Import.organization(
+            {
+                'csv-file': files['users']
+            }
+        )
```

Comment:
I believe this all fits on one line:

```python
rc1 = Import.organization({'csv-file': files['users']})
```

If it does not, try this instead:

```python
rc1 = Import.organization({
    'csv-file': files['users']
})
```

Also, can you please use some variable name other than "rc1"? I have a hard time undrestanding what the variable is, and I suspect others will find the same.